fdbserver/storageserver.actor.cpp

```diff
@@ -177,13 +178,15 @@ struct StorageServerDisk {
 	KeyValueStoreType getKeyValueStoreType() const { return storage->getType(); }
 	StorageBytes getStorageBytes() const { return storage->getStorageBytes(); }
 	std::tuple<size_t, size_t, size_t> getSize() const { return storage->getSize(); }
+	
+	bool canPipelineCommits() const {return storage->canPipelineCommits();}
```

Comment:
`canPipelineCommits()` is a virtual function but its result isn't allowed to change for a given IKVS instance so perhaps just create a bool member in StorageServerDisk to return here which would then be fully inlined.

Reply:
Done.